File: Week2/lab/cmpsc201-fall-21-lab01/src/triangle/java/Triangle.java
import java.util.Scanner;
public class Triangle{
	public static void main(String[] args){
		System.out.print("Enter the number of rows:");
		Scanner scan = new Scanner(System.in);
		int rows = scan.nextInt();
		for (int i =0; i < rows; i++){
			for (int j = 0; j <= i; j++){
				System.out.print("*\t");
			}
			System.out.println();
		}
		
	}
}

File: Week2/lab/cmpsc201-fall-21-lab01/src/triangle/csharp/Triangle.cs
using System;
class Triangle {         
    static void Main(string[] args)
    {
        Console.Write("Enter the number of rows:");
        int rows = Convert.ToInt32(Console.ReadLine());
        for (int i =0; i < rows; i++){
            for (int j = 0; j <= i; j++){
                Console.Write("*\t");
            }
            Console.Write("\n");
        }
    }
}